packages/odyssey/src/scss/components/_checkbox.scss

```diff
@@ -38,7 +38,8 @@ $size-checkbox: 1em / 0.875;
     }
   }
 
-  &:indeterminate {
+  &:indeterminate,
+  &.is-ods-checkbox-indeterminate {
```

Comment:
Since we're already working from within the ods and checkbox namespace, why do we want to namespace it here again?

Reply:
The class names are longer, but we try to namespace everything so that folks immediately know where a class is coming from and what it's scope is. It also helps prevent improper use of various classes.

We're going to take a Long Hard Look at our conventions during Q4 to validate our current setup and move toward better solutions though!

Reply:
Great! That seems like an extremely important decision we should make sooner than later. The longer we take to answer that, the further debt + breaking changes will be required for consumers of this repo.